lib/next_screen.dart

```diff
@@ -0,0 +1,340 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_image_stack/flutter_image_stack.dart';
+
+class NextScreen extends StatefulWidget {
+  const NextScreen({Key? key}) : super(key: key);
+
+  @override
+  State<NextScreen> createState() => _NextScreenState();
+}
+
+class _NextScreenState extends State<NextScreen>{
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor:Color(0xfff6f6f6),
+      body: Container(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          // mainAxisSize: MainAxisSize.max,
+          mainAxisAlignment: MainAxisAlignment.spaceAround,
+          children: [
+            Stack(
+              children: [
+                ListTile(
+                  contentPadding: EdgeInsets.symmetric(horizontal: 10,vertical: 30),
+                  leading: CircleAvatar(
+                    radius: 20,
+                    backgroundColor: Colors.white, // for Right
+                    child: IconButton(onPressed: (){
+                      Navigator.of(context).pop();
+                    }, icon: const Icon(Icons.arrow_back_ios_new_rounded,color: Colors.black,size: 20)),
+                  ),
+                  trailing: const CircleAvatar(
+                      radius: 20,
+                      backgroundColor: Colors.white,
+                      child: Icon(Icons.favorite_border,color: Colors.black,size: 20,)), // for Left
+                ),
+                Align(
+                  alignment: Alignment.topCenter,
+                   child: Image.asset('assets/images/shoes_8.png',
+                    fit: BoxFit.contain,height: 150.0,),
+                ),
+              ],
+            ),
+            SizedBox(height: 5),
+            circlear_Row(),
+            SizedBox(height: 15),
+            main_Container(),
+            SizedBox(height: 10),
+          ],
+        ),
+      ),
+
+
+
+    );
+  }
+}
+
+circlear_Row(){
+  return SingleChildScrollView(
+    scrollDirection: Axis.horizontal,
+    child: Padding(
+      padding: const EdgeInsets.only(left: 5,right: 5),
+      child: Row(
+        // crossAxisAlignment: CrossAxisAlignment,
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: <Widget>[
+          Container(
```

Comment:
#4  make a method for the container because it duplicated many times and then use that method where needed

Reply:
widget getImage(string)